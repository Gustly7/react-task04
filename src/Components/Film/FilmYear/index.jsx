import React, { Component } from 'react'

class FilmYear extends Component {
  render () {
    return (
      <div className='FilmYear'>
        {this.props.filmYear}
      </div>
    )
  }
}

export default FilmYear
