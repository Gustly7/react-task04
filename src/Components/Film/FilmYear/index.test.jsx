import React, { Component } from 'react'
import FilmYear from './index'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('FilmYear component test suite', () => {
  test('FilmYear snapshot renders correctly', () => {
    const tree = renderer.create(
      <FilmYear />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('FilmYear div should have "FilmYear" class', () => {
    const wrapper = shallow(
      <FilmYear />
    )
    const elem = wrapper.find('div.FilmYear')
    expect(elem.length).toBe(1)
  })
})
