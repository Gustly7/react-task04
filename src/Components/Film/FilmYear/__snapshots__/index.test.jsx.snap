// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmYear component test suite FilmYear snapshot renders correctly 1`] = `
<div
  className="FilmYear"
/>
`;
